grpclb/src/main/java/io/grpc/grpclb/GrpclbLoadBalancer.java

```diff
@@ -184,6 +185,11 @@ public void handleNameResolutionError(Status error) {
     }
   }
 
+  @Override
+  public boolean canHandleEmptyAddressListFromNameResolution() {
+    return true;
```

Comment:
oh i may need to fix
https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/internal/ManagedChannelImpl.java#L1388